nixos/modules/tasks/filesystems/zfs.nix

```diff
@@ -871,12 +872,12 @@ in
           Type = "simple";
         };
         script = ''
-          ${cfgZfs.package}/bin/zpool scrub -w ${
+          ${cfgZfs.package}/bin/zpool scrub -w "${
             if cfgScrub.pools != [] then
               (concatStringsSep " " cfgScrub.pools)
             else
               "$(${cfgZfs.package}/bin/zpool list -H -o name)"
-            }
+            }"
```

Comment:
@phaer I think we don't need the quotes around the nix interpolation, do we? It should be sufficient to properly quote the generated bash code.

Could you test if that works? I think you made this change because of your zfs config, right?

Reply:
Sorry, I only saw this notification now. I think your assumption is correct; thanks for the catch @ElvishJerricco. Its just that I am in a train right now and won't be able to properly test this on a zfs machine for a good week, when I return home

Reply:
I dropped this commit in order to unblock this PR, since I don't know zfs well enough to test this.